### ISSUE TITLE
mongodb not installed (correctly) with npm 3.3.10, but with 3.3.9

### ISSUE THREAD

**Issue by manuel-reil** [balderdashy/sails-mongo#359]
Hello,

I saw Issue #338.
Using npm v. 3.3.9, sails-mongo and its dependencies is installed correctly both locally and on heroku via codeship build pipelines.

Now using, npm v. 3.3.10, the dependent package mongodb is not installed in the folder 'sails-mongo/node-modules/mongodb' where it is expected by sails-mongo. 

Sails of course is throwing the error that 'sails-mongo/node-modules/mongodb' cannot be found.
Suprisingly to me, a mongodb folder is located in the node_modules in the sails root folder while I have not specified it in my package.json.

Thank you for any advise or fix.
Manuel

**Comment by sailsbot** [comment]
Thanks for posting, @manuel-reil.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by manuel-reil** [comment]
I still get this error in the most current npm versions like 3.9.0.
Any ideas?

**Comment by nosnibor89** [comment]
I didn't wanted to downgrade my npm version so I used yarn. `yarn add sails-mongo` 

Hope you find it useful...
